src/pybind/mgr/dashboard/controllers/_version.py

```diff
@@ -0,0 +1,75 @@
+import re
+from typing import NamedTuple
+
+
+class APIVersion(NamedTuple):
```

Comment:
@epuertat As this is not a controller but the API versioning logic, I suggest to move this file (without underscore in the name) to the existing: `dashboard/api`.
IMHO it better fits there so we have all API stuff in the same folder.

Reply:
@alfonsomthd I share your point, but the truth is that we currently have the controllers + controller infra mixed in the `controllers` directory (that's why I started using the `_<file>.py` convention in #43137, following what cherrypy does). As this will involve controller instances and related logic (unit tests, integration tests, etc) importing infrastructure logic from 2 different locations (`controller` and `api`), if you are ok I'll defer the move/discussion to the controller clean-up #43137.

Reply:
I'm OK with it.